lib/domain/photo/photo.dart

```diff
@@ -26,12 +28,26 @@ class Photo {
 
   /// Parse json representation to dart photo object.
   factory Photo.fromJson(Map<String, dynamic> json) {
+    Color parsedColor;
+    // Shadow color in json stores in String, code parses it to hex and cast to material Color.
+    try {
+      parsedColor = Color(
+          int.parse(json['color'].toString().substring(1, 7), radix: 16) +
```

Comment:
Как я уже говорил, в таких случаях стоит оставлять пояснения - например:
```dart
/// Since we get shadow color as `0x######`, we have to cut first symbol and add non-transparent alpha channel

Reply:
Также является хорошей практикой хранить подобные преобразования в отдельных классах (конвертерах, мапперах и тд), тк это тоже своего рода логика, которая не должна содержаться в дата-классах